Legobot/Connectors/Discord.py

```diff
@@ -0,0 +1,318 @@
+from websocket import create_connection
+import json
+import logging
+import requests
+import sys
+import threading
+import time
+
+from Legobot.Lego import Lego
+from Legobot.Message import Message, Metadata
+from Legobot.Utilities import Utilities
+
+__author__ = "Bren Briggs (fraq)"
+__copyright__ = "Copyright 2017, Legobot"
+
+logger = logging.getLogger(__name__)
+
+
+class Heartbeat(threading.Thread, object):
+    """
+    Manage heartbeat connection to Discord
+
+    Attributes:
+        ws: Websocket connection to Discord
+        interval (float): time between heartbeat pings to the server
+
+    """
+
+    def __init__(self, ws, interval):
```

Comment:
What is ws? Can we use a more descriptive name?

Reply:
I see later it's marked as web socket in the comments, but I'd like to see the name reflected in the actual code to keep with best practices.